lib/seattleflu/db/cli/command/etl/clinical.py

```diff
@@ -224,28 +225,41 @@ def encounter_details(document: dict) -> dict:
         }
 
 
-def race(race_name: str) -> list:
+def race(races: str or list) -> list:
     """
-    Given a *race_name*, returns the matching race identifier found in Audere
-    survey data.
+    Given one or more *races*, returns the matching race identifier found in
+    Audere survey data.
     """
-    if race_name is None:
+    if races is None:
         LOG.debug("No race response found.")
         return [None]
 
+    if not isinstance(races, list):
+        races = [races]
+
     race_map = {
         "American Indian or Alaska Native": "americanIndianOrAlaskaNative",
+        "amerind": "americanIndianOrAlaskaNative",
         "Asian": "asian",
         "Black or African American": "blackOrAfricanAmerican",
+        "black": "blackOrAfricanAmerican",
         "Native Hawaiian or Other Pacific Islander": "nativeHawaiian",
+        "nativehi": "nativeHawaiian",
         "White": "white",
         "Multiple races": "other",
+        "refused": None,
     }
 
-    if race_name not in race_map:
-        raise UnknownRaceError(f"Unknown race name «{race_name}»")
+    for i in range(len(races)):
+        if races[i] in race_map.values():
+            continue
+
+        if races[i] not in race_map:
+            raise UnknownRaceError(f"Unknown race name «{races[i]}»")
+
+        races[i] = race_map[races[i]]
 
-    return [race_map[race_name]]
+    return races
```

Comment:
To me, this would be more clearly expressed as a [pure function](https://en.wikipedia.org/wiki/Pure_function) combined with `map`, something like:

```python
def standardize_name(name):
    try:
        return name if name in race_map.values() else race_map[name]
    except KeyError:
        raise UnknownRaceError(f"Unknown race name «{name}»") from None

return list(map(standardize_name, races))
```